ui-manchette-with-spacetimechart/src/stories/base.stories.tsx

```diff
@@ -1,7 +1,6 @@
-/* eslint-disable import/no-unresolved */
 import React, { useRef } from 'react';
 
-import { Manchette } from '@osrd-project/ui-manchette';
+import Manchette from '@osrd-project/ui-manchette';
 import type { ProjectPathTrainResult, Waypoint } from '@osrd-project/ui-manchette/dist/types';
```

Comment:
Now that these are exported from `@osrd-project/ui-manchette` (without `dist/`), maybe we could use that instead?

Reply:
Good point, done